frontend/src/metabase/embedding/embedding-hub/utils/embedding-hub-steps.ts

```diff
@@ -33,7 +33,7 @@ export const getEmbeddingHubSteps = (): EmbeddingHubStep[] => {
     },
     actions: [
       {
-        title: t`Add data`,
+        title: t`Connect a database`,
```

Comment:
I don't see this being discussed in the Slack thread. Was this something the design already had, but we forgot to update, or it's something else? Same goes for the `Create a dashboard`

Reply:
@WiNloSt It's newly updated in the design, see [this Slack thread](https://metaboat.slack.com/archives/C07JBRAQ9PA/p1759317572310019?thread_ts=1758646088.331719&cid=C07JBRAQ9PA).